### PR TITLE
Improve 'default after variadic' changelog messages

### PR DESCRIPTION
**Pull request by wilzbach**
The current message: https://dlang.org/changelog/pending.html#default_after_variadic

### CODE REVIEW

File: changelog/default_after_variadic.dd [hunk 1]
```diff
@@ -10,8 +15,10 @@ string log(T...)(T a, string file = __FILE__, int line = __LINE__)
...
 ---
 
-This should be preferred to the previous workaround, which caused template bloat:
+This should be preferred to the previous workaround, which causes a new template instantiation
+for every invocation:
 ---
 string log(string file = __FILE__, int line = __LINE__, T...)(T a);
 ---
 
+For more details, see $(DMDSRC test/runnable/testdefault_after_variadic.d).
```
Comment (jacob-carlborg) [inline changelog/default_after_variadic.dd]: Should we really refer to internal tests in the compiler?

Comment (wilzbach) [reply changelog/default_after_variadic.dd]: This was previously in the title.

Comment (jacob-carlborg) [reply changelog/default_after_variadic.dd]: I know.

Comment (jacob-carlborg) [reply changelog/default_after_variadic.dd]: Why don't we refer to the spec instead?

Comment (wilzbach) [reply changelog/default_after_variadic.dd]: https://github.com/dlang/dlang.org/pull/2169

**Review by jacob-carlborg — APPROVED** [review]

**Pull request merged**